srcs/app/utils/StringUtils.cpp

```diff
@@ -0,0 +1,19 @@
+#include <StringUtils.hpp>
```

Comment:
So, generally, app folder has only startup functions, parsing the command line, and platform dependant code, if you have different entrypoints for different platforms, and then, below it but not inside, lib folder with everything else. In lib, you have every function you could compile into a library and doesn't need nor have a main (.o, .a, .so, .dll, whatever)

something like:

```
srcs/
  srcs/app
    srcs/app/test
  srcs/lib
    srcs/lib/utils
      srcs/lib/utils/test
    srcs/lib/http
      srcs/lib/http/test
    srcs/lib/yourcodegoeshere
      srcs/lib/yourcodegoeshere/test
    .
    .
    .
  srcs/incs
bld/
  .
  .
  .
```
Do we like this idea?

Reply:
We could actually compile the lib srcs to actual libraries, and compile main with those libraries. Like that, when you are in some folder and you need functions from some other folder, in the test, if mocking is not useful, you just add the library you are compiling against, as a higher level abstraction.
As library goes, I like dynamic libraries (.so, .dylib), but since we have to support two platforms, and dynamic link libraries are complex with lots of linker trickery, we could go with static (.a).